src/library_emmalloc.js

```diff
@@ -1,24 +0,0 @@
-/**
- * @license
- * Copyright 2020 The Emscripten Authors
- * SPDX-License-Identifier: MIT
- */
-
-mergeInto(LibraryManager.library, {
-  emmalloc_unclaimed_heap_memory: function() {
-  var dynamicTop = _sbrk();
-#if ALLOW_MEMORY_GROWTH
-#if MAXIMUM_MEMORY != -1
-  // Using MAXIMUM_MEMORY to constrain max heap size.
-  return {{{ MAXIMUM_MEMORY }}} - dynamicTop;
-#else
-  // Not using a Wasm memory bound.
-  return 2*1024*1024*1024 - 65536 - dynamicTop;
```

Comment:
This change seems good, however, the returned value here (2GB - page size) differs from what will be returned from `emscripten_get_heap_max`, which returns 4GB. That is actually a bug - it wraps to 0. It's pre-existing, but this would regress emmalloc, so I think we should fix it first. See #14088